### ISSUE TITLE
Generate changebar document

### ISSUE THREAD

**Issue by barmac** [omg-dmn-taskforce/plain-text-spec#3]
Generate changebar document

**Comment by barmac** [comment]
Tool candidate: https://github.com/davidar/pandiff

**Comment by barmac** [comment]
1. I `pandiff` both files and produce PDF
2. The PDF is ready to view.

**Comment by barmac** [comment]
I run into a problem with `pandoc` which should be solvable:

```
❯ npx pandiff -s origin.adoc 10-expression-language-FEEL.adoc -o diff.pdf
Unknown input format asciidoc
Error: 21
```

**Comment by barmac** [comment]
At https://pandoc.org/, it says that asciidoc is only a target format for the tool. That could explain why it doesn't work. Let's dig more.

**Comment by barmac** [comment]
For some reason the `pandiff` command failed:

```
❯ npx pandiff origin.html 10-expression-language-FEEL.html -o diff.pdf
Error producing PDF.
! LaTeX Error: File `soul.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name:
! Emergency stop.
<read *>

l.45 \fi

Error: 43
```

Gotta look into this later.

**Comment by barmac** [comment]
Output of this command looks terrible:

`npx pandiff origin.html 10-expression-language-FEEL.html -o diff.html`

File needs to be renamed to `diff.html`:

[diff.html.txt](https://github.com/omg-dmn-taskforce/plain-text-spec/files/12004321/diff.html.txt)

**Comment by falko** [comment]
Yes, just strike-through and underline is not very pretty. Also we would need to annotate, by which issue/PR a change has been made.
